kimchi/js/bindings.js

```diff
@@ -2808,7 +2670,8 @@ var caml_pasta_fq_proof_of_rust = function (x) {
     res[1] = tsRustConversion.fq.vectorFromRust(
       prev_challenges_scalars.get(i - 1)
     );
-    res[2] = caml_pallas_poly_comm_of_rust(prev_challenges_comms[i]);
+    // FIXME (gregor): this accesses prev_challenges_comms at one index too high
+    res[2] = tsRustConversion.fq.polyCommFromRust(prev_challenges_comms[i]);
```

Comment:
Ah? Already before?

Reply:
I think so, but no need to figure this out here IMO since one of the next commits moves it to TS anyway and it's much easier to reason about these things in the new code (where the `[0, ..array]` structure is encoded in types)